app/pages/Settings/index.js

```diff
@@ -195,7 +222,22 @@ export default class Settings extends Component {
           null,
           walletSync,
         )}
-
+        {this.renderSection(
+          'Backup WalletDB',
+          <div>
+            <div>{`Back up wallet database and files to a directory.`}</div>
+          </div>,
+          'Backup WalletDB',
+          this.onBackupWDB,
+        )}
+        {this.renderSection(
+          'Deep Clean and Rescan Wallet',
+          <div>
+            <div>For more information on deep clean, please see <Anchor href="https://github.com/handshake-org/hsd/blob/master/CHANGELOG.md#wallet-api-changes-1">here</Anchor></div>
```

Comment:
This link may change as new versions get added to the hsd changelog -- I'm not sure a good static link to include here, maybe the PR? https://github.com/handshake-org/hsd/pull/511